### ISSUE TITLE
Test mappings work properly after schema update

### ISSUE THREAD

**Issue by mike-podolskiy90** [gbif/ipt#2226]
After schema update there might be issues with mappings. Cases:

- [ ] New field added to schema
- [ ] Field removed from schema
- [ ] Field type changed in schema
- [ ] New subschema added
- [ ] Subschema removed (renamed)

**Comment by vechocho** [comment]
This is related to [#2095](https://github.com/gbif/ipt/issues/2095) ?

**Comment by mike-podolskiy90** [comment]
@vechocho No, it's not. This issues is about the Frictionless Data Package update, not DwC.
